src/routes/users.js

```diff
@@ -13,7 +13,6 @@ router.get(
 router.post('/', userController.createUser);
 router.put(
     '/:id',
-    [hasPermissions('user-management')],
```

Comment:
@KenyerRamirez shouldn't be this endpoint protected  I mean if the user was created, shouldn't be the user it self the one that could update its profile picture?
is there any user role for the regular users trying to update its profile?

Reply:
Not only its picture, the other data too, I mean: name, description, contact, etc. You're right, the middleware protect the endpoint, but this middleware doesn't let the user update it own data, for that reason I deleted that line.

Reply:
@PedroLabrador is there any permission for the same user I wont't like users profile picture be allow to updates without being the owner user itself

Reply:
what I mean is shouldn't be something like:
hasPermissions('user-management') || hasPermissions('user-owner'),